File: src/main/scala/com/bombproj/vo/ProjectCodeVO.java
package com.bombproj.vo;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

import java.util.Date;

@Data
public class ProjectCodeVO {

    private String id;
    private String code;
    private String message;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm")
    private Date createTime;

}

File: src/main/scala/com/bombproj/vo/ProjectCodeVO.scala
package com.bombproj.vo

import java.util.Date

import com.fasterxml.jackson.annotation.JsonFormat

import scala.beans.BeanProperty

case class ProjectCodeVO() {

    @BeanProperty var id: String = _
    @BeanProperty var code: String = _
    @BeanProperty var message: String = _
    @BeanProperty @JsonFormat(pattern = "yyyy-MM-dd HH:mm") var createTime: Date = _

}
